adapters/AppLovin/AppLovinAdapter/RTB/GADMAppLovinRtbRewardedRenderer.h

```diff
@@ -0,0 +1,24 @@
+//
+//  GADMAppLovinRtbRewardedRenderer.h
+//  Adapter
+//
+//  Created by Christopher Cong on 7/17/18.
+//  Copyright © 2018 Google. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface GADMAppLovinRtbRewardedRenderer : NSObject
+
+- (void)loadAd;
+
+- (instancetype)initWithAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
+                      completionHandler:(nonnull GADRewardedRenderCompletionHandler)handler;
```

Comment:
What's the point of specifying `nonnull` when we are already assuming it?

Reply:
Copied from AdMob, will remove.